service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentTaskExecutor.java

```diff
@@ -75,7 +75,7 @@ private EhrExtractStatus handleValidGpcDocument(String response, GetGpcDocumentT
         storageConnectorService.uploadFile(storageDataWrapperWithMhsOutboundRequest, documentName);
 
         return ehrExtractStatusService.updateEhrExtractStatusAccessDocument(
-            taskDefinition, documentName, taskId, taskDefinition.getMessageId(), contentAsBase64.length());
+            taskDefinition, documentName, taskId, taskDefinition.getMessageId(), contentAsBase64.length(), null);
```

Comment:
Should add assertion that this is indeed NULL in the case of a successful document download.